src/js/services/forms.js

```diff
@@ -136,31 +136,36 @@ export default App.extend({
       });
     });
   },
-  _getPrefillFilters(flowId, form) {
+  _getPrefillFilters({ actionId, flowId, patientId }, form) {
     const prefillActionTag = form.getPrefillActionTag();
 
     if (prefillActionTag) {
       return {
+        'action': actionId,
         'action.tags': prefillActionTag,
```

Comment:
The combination of `flow + action + action.tags` is illogical, because we cannot determine if `action` or `action.tags` should take precedence, given that `action` may not have `action.tags` at all.

Is the assumption that either `flow` or `action` will be filtered out because they have empty values?

Reply:
hmm yeah.. I'm actually not sure what use case we'd have for `action` right now.  Really I should have been doing this for patient, so I switched to that.. which I _think_ makes this essentially backwards compatible with how it's currently working